.gitignore

```diff
@@ -102,3 +102,5 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+\.idea/
```

Comment:
Why the backslash?

Reply:
not certain, it was autogenerated by gitkraken. Does it change the meaning?

Reply:
It doesn't mean anything, which is why none of the lines above have it
```suggestion
# PyCharm project settings
.idea/
```